internal/person/repository.go

```diff
@@ -63,14 +63,13 @@ func (repo *personRepository) CreateProfile(ctx context.Context, iamID string, p
 	if err != nil {
 		if err == sql.ErrNoRows {
 			person = &models.Person{
-				IamID:          iamID,
-				FirstName:      null.StringFrom(params.FirstName),
-				LastName:       null.StringFrom(params.LastName),
-				CurrentCompany: null.StringFrom(params.Company),
-				Email:          params.Email,
-				IamProvider:    "LinkedIn",
-				AvatarURL:      null.StringFrom(params.AvartarURL),
-				ID:             u2.String(),
+				IamID:       iamID,
+				FirstName:   null.StringFrom(params.FirstName),
+				LastName:    null.StringFrom(params.LastName),
+				Email:       params.Email,
+				IamProvider: "LinkedIn",
+				AvatarURL:   null.StringFrom(params.AvartarURL),
```

Comment:
typo: should be `AvatarURL`

Reply:
also, typo at this line:
https://github.com/jobsgowhere/jobsgowhere/blob/210bf3dcb705f31efae709354bae1028c95e4711/internal/person/model.go#L10

Reply:
@subhransu raising this issue again :)

Reply:
This is outdated I think. Can you check the latest commit? :)

Reply:
Yes it is! Was looking at an outdated commit 👍